### PR TITLE
Add a service config guide

### PR DESCRIPTION
**Pull request by temawi**

### CODE REVIEW

File: content/en/docs/guides/service-config.md [hunk 1]
```diff
@@ -8,72 +8,70 @@ description: >-
...
 
 #### Load Balancing
 
 A service can be composed of multiple servers and the load balancing
-configuration how calls from client will be distributed among those servers.
-By default the `pick_first` load balancing policy is utilized, but another
-one can be specified in the service config. E.g. by specifying the
+configuration specifies how calls from clients should be distributed among
+those servers. By default the `pick_first` load balancing policy is utilized,
+but another policy can be specified in the service config. E.g. by specifying the
```
Comment (ejona86) [inline content/en/docs/guides/service-config.md:27]: s/by specifying/specifying/ (or adjust "will" on the next line)

File: content/en/docs/guides/service-config.md [hunk 2]
```diff
@@ -8,72 +8,70 @@ description: >-
...
 `round_robin` policy will make the clients rotate through the servers when
-calling the service instead of repeatedly using the first server they know of.
+calling the service instead of repeatedly using the first server.
```
Comment (ejona86) [inline content/en/docs/guides/service-config.md:29]: Could we just remove "when calling the service"? Is that cleaner?

File: content/en/docs/guides/service-config.md [hunk 3]
```diff
@@ -8,72 +8,70 @@ description: >-
...
 
-The way remote calls are made can be configured in many ways:
+RPCs can be configured in many ways:
 
-- [Wait-for-ready] can be enabled, which will instruct the client to wait for
-  the backend to become available instead of failing a call.
-- A call [timeout] can be provided, indicating the maximum time the client is
-  willing to wait for a server to respond.
+- With [wait-for-ready] enabled, if a client cannot connect to a backend, the
+  RPC will be delayed instead of immediately failing. a backend to become
```
Comment (ejona86) [inline content/en/docs/guides/service-config.md:36]: "a backend to become..." What's going on there?

File: content/en/docs/guides/service-config.md [hunk 4]
```diff
@@ -8,72 +8,70 @@ description: >-
...
+that service owners can use to distribute their service config out to a fleet
+of gRPC clients.
 
-- The DNS name resolver that comes with gRPC clients supports service configs
-  [stored as TXT records] on the name server.
-- The xDS name resolver used with a control plane server in a microservices
-  architecture uses the responses it gets from the control plane to construct
-  the service config.
+- The xDS name resolver converts the xDS configuration it receives from the
+  control plane to a corresponding service config
```
Comment (ejona86) [inline content/en/docs/guides/service-config.md:72]: Add a period for consistency with the next bullet?

File: content/en/docs/guides/service-config.md [hunk 5]
```diff
@@ -86,17 +84,18 @@ prefer as long as they provide it in JSON format at name resolution time.
 #### Programatically
 
 The gRPC client API provides a way to specify a service config in JSON format.
-This is used to provide default values for settings that the name resolver might
-not provide values for. It can also be useful in some testing situations.
+This is used to provide a default service config that will be used in situation
```
Comment (ejona86) [inline content/en/docs/guides/service-config.md:87]: s/situation/situations/ ? or "the situation"?

**Review by ejona86 — APPROVED** [review]
This looks fine to me, but I think we need to say somewhere "this is scoped to a Channel" or "this is scoped to a target string" so that it isn't misunderstood when we say things like "all methods in all services"

**Comment by temawi** [comment]
> This looks fine to me, but I think we need to say somewhere "this is scoped to a Channel" or "this is scoped to a target string" so that it isn't misunderstood when we say things like "all methods in all services"

I added this to the overview section:

"The settings in a service config always apply
to a specific target string (e.g. "api.myapp.com"), not globally."

**Pull request merged**